src/utils/parse.ts

```diff
@@ -0,0 +1,52 @@
+export function parseRecipe(recipe: GatsbyTypes.ContentfulRecipe) {
+  const totalTime =
+    recipe?.prepTime && recipe?.totalTime ? recipe?.prepTime + recipe?.totalTime : '';
+
+  const prepDirections = recipe?.prepDirections?.prepDirections
+    ?.split('\n')
+    .map(prepDir => prepDir.slice(2));
+
+  const instructions = recipe?.directions?.directions
+    ?.split('\n')
+    .map(direction => direction.slice(2));
+
+  const notes = recipe?.notes?.notes?.split('\n').map(note => note.slice(2));
```

Comment:
```suggestion
  const notes = recipe?.notes?.notes?.split('\n').map(note => note.slice(2)) ?? [];
```

Fixes the aforementioned linting issue